node-module-system/calculate.js

```diff
@@ -0,0 +1,16 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv[3] === 'plus') {
+  add(+process.argv[2], +process.argv[4]);
+} else if (process.argv[3] === 'minus') {
+  subtract(process.argv[2], process.argv[4]);
```

Comment:
Technically, you should convert all of these to number, not just `add`. You are taking advantage of the (unfortunate) behavior of JavaScript to convert strings to numbers behind your back when it can. It's best to be explicit.